### PR TITLE
cmd: add ability to sync projects for release process

### PR DESCRIPTION
**Pull request by aanm**
Usage:
`./release --current-version 1.9.2 --next-dev-version 1.9.3`

Signed-off-by: André Martins <andre@cilium.io>

### CODE REVIEW

File: cmd/projects/projects.go [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+		if proj.GetName() == curr {
+			currentProjID = proj.GetID()
+		}
+		if proj.GetName() == next {
+			nextProjID = proj.GetID()
+		}
+	}
+	return currentProjID, nextProjID, nil
+}
+func (pm *ProjectManagement) findColumnIDs(ctx context.Context, projID int64, ver string) (needs, pending, done int64, err error) {
```
Comment (christarazi) [inline cmd/projects/projects.go:73]: Looks like need to run `gofmt`?

File: cmd/projects/projects.go [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+	return &ProjectManagement{
+		owner:    owner,
+		repo:     repo,
+		ghClient: ghClient,
+	}
+}
+
+// currVer, nextVer := "1.9.2", "1.9.3"
+func (pm *ProjectManagement) SyncProjects(ctx context.Context, currVer, nextVer string) error {
+
```
Comment (christarazi) [inline cmd/projects/projects.go:236]: Nit: Extra line

File: cmd/projects/projects.go [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+	err = pm.syncCards(ctx, currVer, nextVer, currPendingColumnID, nextPendingColumnID, currDoneColumnID, nextPendingColumnID)
+	if err != nil {
+		return err
+	}
+
+	// Close the current project when we change the order of release process
+	// fmt.Fprintf(os.Stdout, "Closing project %q\n", currVer)
+	// _, _, err = pm.ghClient.Projects.UpdateProject(ctx, currProjID, &gh.ProjectOptions{
+	// 	State: func() *string { a := "closed"; return &a }(),
+	// })
```
Comment (christarazi) [inline cmd/projects/projects.go:316]: Is this needed anymore?

**Review by christarazi — COMMENTED** [review]
Nice! This is going to save so much time :rocket: :tada:

File: cmd/projects/projects.go (hunk 3 above)
Comment (aanm) [reply cmd/projects/projects.go]: Yes, I've added a better TODO message.

**Review by christarazi — APPROVED** [review]

**Pull request merged**